src/data/metric.ts

```diff
@@ -18,7 +18,13 @@ export function counter(metric: Metric, increment: number = 1): void {
     track(metric);
 }
 
-export function measure(metric: Metric, value: number): void {
+export function duration(metric: Metric, time: number): void {
```

Comment:
>duration [](start = 16, length = 8)

Suggest accumulate as better name for this.

Reply:
Done